### PR TITLE
Add doc on rotating full RP env VPN certs

### PR DESCRIPTION
**Pull request by cadenmarchese**
### Which issue this PR addresses:

n/a

### What this PR does / why we need it:

Adds doc on how to rotate certificates on a vnet gateway created as part of the full RP env setup. I've had to do it a few times so I figured it would be useful to document it upstream.

### Test plan for issue:

n/a, docs issue.

### CODE REVIEW

**Review by mociarain — DISMISSED** [review]
LGTM

**Review by kimorris27 — APPROVED** [review]

File: docs/deploy-full-rp-service-in-dev.md [hunk 1]
```diff
@@ -398,3 +398,38 @@
...
+
+Since setting up your own VPN in an earlier step will overwrite your local secrets, you will lose access to the vpn / vnet gateway that you provisioned in an earlier step if you run `make secrets`. If you don't have a secrets/* backup, you can recover your access using the following steps. Please note that this action will _**OVER WRITE**_ the `secrets/vpn-$LOCATION.ovpn` on your local machine. **DO NOT** run `make secrets-update` after doing this, as you will overwrite the shared secrets for all users.
+
+1. Source all environment variables from earlier, and run the VPN configuration step again:
+
+    ```bash
+    . ./env
+    . ./env-int
+
+    source ./hack/devtools/deploy-shared-env.sh
```
Comment (s-fairchild) [inline docs/deploy-full-rp-service-in-dev.md:412]: Nit pick, I would prefer either `.` or `source` is used, rather than a mix of the two.

**Review by s-fairchild — APPROVED** [review]
Small discrepancy, otherwise LGTM.

File: docs/deploy-full-rp-service-in-dev.md (hunk 1 above)
Comment (cadenmarchese) [reply docs/deploy-full-rp-service-in-dev.md]: done!

**Review by kimorris27 — APPROVED** [review]

**Pull request merged**